src/avram/validations.cr

```diff
@@ -94,12 +94,13 @@ module Avram::Validations
   #
   # This will mark `state` as invalid unless the value is `"NY"`, or `"MA"`.
   def validate_inclusion_of(
-    attribute : Avram::Attribute(T),
+    attribute : Avram::Attribute,
```

Comment:
Was removing this required? The nice thing about leaving it is that it enforces the allowed values to be the same type as the attribute. If we can change this back that’d be awesome!

Reply:
That would be a missed opportunity. Still learning a lot about the type system. :)